docs/user-tests/user-test-02-Template-EntwicklerInnen.md

```diff
@@ -0,0 +1,60 @@
+# User-Test für EntwicklerInnen
+
+**Testperson:**
+
+**Datum:**
+
+## Aufgaben
+
+Folgende Aufgaben sollen im User-Test vorgenommen werden, getestet werden kann jedoch noch mehr. 
+Sofern daneben noch weitere Funktionen getestet wurden, würden wir Sie darum bitten, diese in der Tabelle zu vermerken.
+
+1) Bilder hinzufügen
+2) Bilderebenen auswählen/löschen/hinzufügen
+3) Zoomen
+4) Transparentseinstellungen für jede Ebene ändern
+5) Einstellungen der Bilder ändern
+6) Komponenten hinzufügen/löschen
+7) Design und Farbe von der Anwendung ändern
+8) Einfügen eigener Bilder in die Galerie
+9) Anpassung des Textes (keine Bilder ausgewählt)
+10) Nutzung aller Elemente ohne Galerie 
+
+
+## User-Test 
```

Comment:
- [ ] Beispielprojekt herunterladen (clonen)
- [ ] Vue-Komponente cross-fade-viewer (CFV) installieren
- [ ] CFV in UI einbinden
- [ ] Einfügen eigener Bilder (Bild-URLs)
- [ ] Theme auswählen
- [ ] Anpassen des "Keiner Bilder da"-Textes
- [ ] Ausblenden der Galerie